app/views/hosts/_dhcp_lease_errors.html.erb

```diff
@@ -15,7 +15,7 @@
         </div>
       </div>
       <div class="modal-footer">
-        <button type="button" class="btn btn-default" data-dismiss="modal"><%= _('ACK') %></button>
+        <button type="button" class="btn btn-default" data-dismiss="modal"><%= _('OK') %></button>
```

Comment:
Because afaik, it will not create an overwrite, as it is an error and not a conflict - It will ask you to fix the errors (also implied with the modal message: `Please correct the error(s) below and submit your changes again.`
Whereas, in conflict you may overwrite it, hence the change in the button text. (also note the modal message in conflict is different)